zipkin-core/src/main/scala/org/http4s/zipkin/core/interpreters/collector/Http.scala

```diff
@@ -0,0 +1,41 @@
+package org.http4s.zipkin.core.interpreters.collector
+
+import argonaut.Argonaut._
+import org.http4s.argonaut._
+import org.http4s.Uri.RegName
+import org.http4s.client.Client
+import org.http4s.zipkin.core.algebras.Collector
+import org.http4s.zipkin.core.ZipkinInfo
+import org.http4s.{MediaType, Method, Request, Uri}
+
+import scalaz.concurrent.Task
+
+final case class Http(client: Client) extends Collector {
```

Comment:
note if you want, and aren't thrown by it, there's a ZipkinRule test server you can use for interop testing.

https://github.com/openzipkin/zipkin/tree/master/zipkin-junit#usage